apps/src/code-studio/videos.js

```diff
@@ -228,8 +228,13 @@ videos.showVideoDialog = function(options, forceShowVideo) {
   nav.append(fallbackPlayerLinkDiv);
 
   // Resize modal to fit constraining dimension.
-  var height = $(window).height() * widthRatio,
-    width = $(window).width() * heightRatio;
+  let accurateInnerHeight = Math.min(window.innerHeight, window.outerHeight);
+  let accurateInnerWidth = Math.min(window.innerWidth, window.outerWidth);
+  let windowHeight = Math.min(accurateInnerHeight, accurateInnerWidth);
+  let windowWidth = Math.max(accurateInnerHeight, accurateInnerWidth);
```

Comment:
Android reports inconsistent values for innerHeight and innerWidth. This workaround accounts for that issue.
<img src="https://user-images.githubusercontent.com/8324574/78621663-fc522e00-7837-11ea-8e3c-b5abc9c5e727.jpg" width="200"/>
![Screenshot_20200406-135157_Chrome](https://user-images.githubusercontent.com/8324574/78621665-fe1bf180-7837-11ea-9df2-e39023db13c9.jpg)